_sass/main.scss

```diff
@@ -49,8 +51,38 @@ figcaption {
     text-align:center;
 }
 
+figure {
+    padding-bottom: 0.5rem;
+}
+
 // FOOTER
 
+@media screen and (max-width: $small) {
+
+    .footer-col {
+        
+    }
+    .footer-col-1 {
+
+    }
+    .footer-col-2 {
+        display: none;
+    }
+    .footer-col-3 {
+        text-align: left !important;
+    }
+    .social-links li:first-child {
+        padding-bottom: 0.5rem;
+    }
+
+}
+
+@media screen and (min-width: $small) {
```

Comment:
I think this can be wedged inside 
`   .footer-col {

    }`
    
    for hygiene?